crates/x/src/lint.rs

```diff
@@ -91,5 +91,34 @@ pub fn run(args: Args) -> crate::Result<()> {
 
     let results = engine.run()?;
 
-    handle_lint_results(results)
+    handle_lint_results_exclude_external_crate_checks(results)
+}
+
+/// Define custom handler so we can skip certain lints on certain files. This is a temporary till we upstream this logic
+pub fn handle_lint_results_exclude_external_crate_checks(
```

Comment:
Here we are disabling the requirement for Move files to have the license headers
I spoke to Brandon to ensure it was okay before doing this.